graphqlapi/src/main/java/io/stargate/graphql/web/GraphqlCache.java

```diff
@@ -128,7 +128,7 @@ private static String findDefaultKeyspace(DataStore dataStore) {
   }
 
   private void addOrReplaceDmlGraphql(
-      String keyspaceName, String reason, String... reasonArguments) {
+      String keyspaceName, String reason, Object... reasonArguments) {
```

Comment:
Narrowing the type does not seem like a good direction. Why error-prone enforce it?